tools/build_release_uvision.bat

```diff
@@ -15,22 +15,40 @@
 :: See the License for the specific language governing permissions and
 :: limitations under the License.
 ::
+
+@if exist .gitignore goto label0
+	@echo Error: working directory must be root of DAPLink repository
+	@exit /B 1
+
+:label0
+
 rmdir /q /s uvision_release
+@if %errorlevel% neq 0 exit /B %errorlevel%
+
 mkdir uvision_release
+@if %errorlevel% neq 0 exit /B %errorlevel%
+
+@git --version 2> nul
+@if %errorlevel% neq 0 echo Error: git must be in PATH && exit /B %errorlevel%
+
 git rev-parse --verify HEAD                     > uvision_release\git_info.txt
 git diff --no-ext-diff --quiet --exit-code      >> uvision_release\git_info.txt
+
 echo Uncommitted Changes: %errorlevel%          >> uvision_release\git_info.txt
+
 virtualenv env
 call env\Scripts\activate
-REM use project requirements if not specified
+
+@REM use project requirements if not specified
 if [%1]==[] pip install -r requirements.txt
-REM use custom requirements if specified
+@REM use custom requirements if specified
 if not [%1]==[] pip install -r %1
 pip freeze                                      > uvision_release\build_requirements.txt
-REM build but continue if there are errors
+
+@REM build but continue if there are errors
 progen generate -t uvision -b
-SET LEVEL=%ERRORLEVEL%
+@SET level=%errorlevel%
```

Comment:
Just wanted to make sure there wasn't a subtle bug that this fixed.  (I don't do much batch scripting so I could have easily created a bug).  Case doesn't matter to me either.  Keeping everything consistent as you did is preferred.